readme.rst

```diff
@@ -174,6 +174,20 @@ Add custom configuration
 To add custom configurations for your project, see the ``Additions to default configuration`` and ``Additional configuration`` sections in the ``custom_conf.py`` file.
 These can be used to extend or override the common configuration, or to define additional configuration that is not covered by the common ``conf.py`` file.
 
+(Optional) Synchronise GitHub issues to Jira
+--------------------------------------------
+
+If you wish to sync issues from your documentation repository on GitHub to your
+Jira board, configure the `gh-jira-sync-bot <https://github.com/canonical/gh-jira-sync-bot>`_
+by editing the ``.github/workflows/.jira_sync_config.yaml`` file appropriately.
+In addition to updating this file, you must also apply server configuration
+for this feature to work. For more information, see `server configuration details <https://github.com/canonical/gh-jira-sync-bot#server-configuration>`_
+for the gh-jira-sync-bot.
+
+By default, this file in the starter pack contains configuration for syncing
+issues from the starter pack repository to its documentation Jira board and
+hence does not work out of the box for other repositories in GitHub.
```

Comment:
How about:
```suggestion
The ``.jira_sync_config.yaml`` file that is included in the starter pack contains configuration for syncing
issues from the starter pack repository to its documentation Jira board.
Therefore, it does not work out of the box for other repositories in GitHub, and you must update it if you want to use the synchronisation feature.
```

Reply:
By exposing real Doc values in the YAML file, isn't there the risk of users of remote repositories interacting with the Doc Jira board?

Reply:
Not by accident because you need the configuration on the GitHub repo server side for it to work. So just including the default file will not do anything.
There might be an issue if someone includes the default file, does the server configuration for the Canonical Jira, but does NOT change the file. But in that case, they clearly want it to work, so they would probably track it down quickly.